### ISSUE TITLE
p.name() should just be p.name?

### ISSUE THREAD

**Issue by getzdan** [ipython-contrib/IPython-notebook-extensions#376]
in setup.py:24
https://github.com/ipython-contrib/IPython-notebook-extensions/blob/master/setup.py#L24

using linux, this line generated an error fixed by dropping the ().

**Comment by jfbercher** [comment]
Hi, 
I don't think so; it is actually p.name() that returns the name of the process (and p.name the corresponding method). However, after checking this, I saw  in my linux box that "jupyter-notebook" is cut as "jupyter-noteboo" (the trailing "k" is missing). As a consequence the running process is not detected and this may imply an error after. Something like
```
for p in psutil.process_iter():
    if "jupyter-note" in p.name():
```
may solve the issue

**Comment by getzdan** [comment]
After doing another test at the python prompt and getting an error. I believe it is an implementation problem of psutil in linux/python3. If anyone is testing the installation in Linux platform (except for me ;) ), he may have some input. 

Anyways, the proper way to go is having a 3 platform (windows,linux,mac) test done automatically.

**Comment by manx183** [comment]
I ran into the same issue getzdan mentioned using a Ubuntu 14.04.1 VM and removing the () allowed the script to run to completion. I tried editing the "jupyter-note" but still received the same error - 

```Python
Traceback (most recent call last):
  File "setup.py", line 24, in <module>
    if "jupyter-notebook" in p.name():
TypeError: 'str' object is not callable*

*Traceback (most recent call last):
  File "setup.py", line 24, in <module>
    if "jupyter-note" in p.name():
TypeError: 'str' object is not callable*

```
The code that worked was - 
```Python
*for p in psutil.process_iter():
    if "jupyter-note" in p.name:
        print("Cannot install while the Jupyter notebook server is running")
        exit()*
```

**Comment by jfbercher** [comment]
Let's try to check what happens: can you try this and post the first lines of the output?
```python
import psutil
print("psutil version:", psutil.__version__)
for p in psutil.process_iter():
    print("Type of p.name", type(p.name))
    try:
        print("Is p.name callable?", p.name())
    except:
        print("p.name is not callable")
```

**Comment by focus-shift** [comment]
Hi,
       I'm running Kubuntu 14.04 (trusty) with the default trusty python3-psutil (which is psutil version 1.2.1) well and I have the same problem. Replacing line 33 of setup.py with
```python
if "jupyter-notebook" in p.name or any("jupyter-notebook" in c for c in p.cmdline):
```
fixed the problem for me.

**Comment by juhasch** [comment]
Fixed by #462
